examples/hosted-wallet-kt/pfi/build.gradle.kts

```diff
@@ -28,7 +21,16 @@ dependencies {
     implementation("com.sparkjava:spark-core:2.9.4")
     implementation("com.squareup.okhttp3:okhttp:4.12.0")
 
-    implementation("xyz.block:tbdex:commit-d6edc85-SNAPSHOT")
+    // For local development dependency.
+    // Install the dependency locally by running `mvn install` in the `bound/kt` directory
+    // implementation("xyz.block:tbdex:0.0.0-main-SNAPSHOT")
+
+    // For a snapshot from maven
+    // Update the short git commit SHA below
+    // implementation("xyz.block:tbdex:commit-d6edc85-SNAPSHOT")
+
+    // For the official release on maven central
+    implementation("xyz.block:tbdex:3.0.0")
```

Comment:
I want to change this and the other gradle build file to use the latest snapshot, once that's deployed, whereafter we can merge this PR (assuming ✅)

Reply:
this is done